src/util.rs

```diff
@@ -15,12 +15,18 @@ use cbor::CborType;
 pub fn get_sig_struct_bytes(
     protected_body_header_serialized: CborType,
     protected_signature_header_serialized: CborType,
+    external_aad: &[u8],
```

Comment:
I feel like we should just not support aad at all and hard-code the associated value in the signature array to `CborType::Null`.

Reply:
Agreed, we don't need `aad` at the moment.